docs/use-mainnet/bridges-of-linea/how-to-bridge-erc20-tokens.mdx

```diff
@@ -42,6 +43,14 @@ We recommend that only tech operators that are providing liquidity use our bridg
 
 6. You now can bridge this token, and it will be saved in your token list for future bridging.
 
+:::note
```

Comment:
I think this note should go at the top instead of being buried between screenshots

Reply:
included in the note at the top of the page.